Clarify that events are copied into each subscribers input queue (#5290)
Particular/docs.particular.net: nservicebus/messaging/publish-subscribe/index.md

Note: For simplicity these explanations refer to specific endpoints as "Subscribers" and "Publishers". However in reality any endpoint can be both a publisher and/or and a subscriber.

> ### All subscribers gets a copy of the event

Reviewer: ```suggestion
### All subscribers get a copy of the event
```

> ### All subscribers gets a copy of the event

Reviewer 2: ```suggestion
### All subscribers gets their own copy of the event
```

> To make sure that each subscriber can process, and potentially retry, the event independent from other subscribers NServiceBus makes sure that each subscriber gets a copy of the published event delivered to their input queue.

Reviewer: ```suggestion
To make sure that each subscriber can process, and potentially retry, the event independent of other subscribers NServiceBus makes sure that each subscriber gets a copy of the published event delivered to their input queue.
```
